src/Docs/Resources/current/4-how-to/270-custom-module.md

```diff
@@ -157,6 +157,23 @@ navigation: [{
 }]
 ```
 
+### Settings
+
+You can provide custom setting cards that are either placed in shop or system.
+This can be easily achieved by adding the key `settingsItem` to your module object:
+
+```js
+settingsItem: [{ /* this can be a single object if no collection is needed */
+    to: 'custom.module.overview', /* route to anything */
+    group: 'system', /* either system or shop */
```

Comment:
isn't plugin also possible? 🤔

Reply:
I am unsure how to interpret this correctly https://github.com/shopware/platform/blob/cb6f8c816cde206389f6a4d4877bd17e3279e1d8/src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/sw-settings-index.html.twig#L80

Reply:
@lernhart why will the plugin part be removed?

Reply:
@mitelg it won't be removed at all
We changed the way, how you have to register your plugin's settings to show up in the administration settings page
Before, you needed to extend the twig template and render your settings item manually in the plugins slot (this is deprecated for v6.4.0 now) 

From v6.4.0 on, the slot will be rendered automatically

Reply:
So I can now just add setting groups myself or am I still limited to system, shop, plugin?

Reply:
For this, the template gets the items from the newly creates settings-items store
In the store, all settings items are grouped by the "group" key defined in your Module's  settingsItem configuration, each representing a tab on the left side of the settings administration index page

We only render the "plugins" slot here manually, to ensure, that plugin manufacturers have at least time to 6.4.0 to adjust their plugins to the new configuration

In 6.4.0, a slot for each of the groups in the settingsItems store will be rendered automatically

See:
https://github.com/shopware/platform/blob/cb6f8c816cde206389f6a4d4877bd17e3279e1d8/src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/sw-settings-index.html.twig#L53

Reply:
@JoshuaBehrens it should be possible if you define a own tab

https://github.com/shopware/platform/blob/cb6f8c816cde206389f6a4d4877bd17e3279e1d8/src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/sw-settings-index.html.twig#L14

Reply:
Thanks for clarifying that. I'll mention that in the docs.

Reply:
Thank you for your contribution :+1: